lib/iris/tests/unit/test_Future.py

```diff
@@ -57,6 +51,14 @@ def test_invalid_netcdf_promote_attributes(self):
             with self.assertRaisesRegexp(AttributeError, exp_emsg):
                 future.netcdf_promote = state
 
+    def test_invalid_netcdf_no_unlimited_attributes(self):
+        future = Future()
+        states = [True, False]
+        exp_emsg = ''
+        for state in states:
+            with self.assertRaisesRegexp(AttributeError, exp_emsg):
```

Comment:
You can just use `self.assertRaises()` if you don't want to specify a message.

Reply:
No, that means I forgot to add the error message...